### ISSUE TITLE
Pypi package for release 0.4.3 seems not correct

### ISSUE THREAD

**Issue by guilleJB** [novafloss/workalendar#130]
Hi, 

I'm try to install workalendar using `pip` but when I'm run `pip install workalendar` it shows a next error:
```bash
Downloading/unpacking workalendar from https://pypi.python.org/packages/12/86/c1fc76cf70bf66a134f40ac8c1c1398f5966ba9681b451055f2ae0a97e48/workalendar-0.4.3.linux-x86_64.tar.gz#md5=4b1d4cf26d3477b78c42aa052bf897ab
  Downloading workalendar-0.4.3.linux-x86_64.tar.gz (50kB): 50kB downloaded
  Running setup.py egg_info for package workalendar
    Traceback (most recent call last):
      File "<string>", line 16, in <module>
    IOError: [Errno 2] No such file or directory: '/home/gjulia/ve/erp/build/workalendar/setup.py'
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):

  File "<string>", line 16, in <module>

IOError: [Errno 2] No such file or directory: '/home/gjulia/ve/erp/build/workalendar/setup.py'

----------------------------------------
Command python setup.py egg_info failed with error code 1 in /home/gjulia/ve/erp/build/workalendar
Storing complete log in /home/gjulia/.pip/pip.log
```

My pip version is 1.3.1

So I'm download next link using wget 

`https://pypi.python.org/packages/12/86/c1fc76cf70bf66a134f40ac8c1c1398f5966ba9681b451055f2ae0a97e48/workalendar-0.4.3.linux-x86_64.tar.gz#md5=4b1d4cf26d3477b78c42aa052bf897ab`  

Then when I decompressed the file it seems that is not ok:

```bash
#tar -zxvf workalendar-0.4.3.linux-x86_64.tar.gz 
./
./home/
./home/novapost/
./home/novapost/.virtualenvs/
./home/novapost/.virtualenvs/TOX/
./home/novapost/.virtualenvs/TOX/lib/
./home/novapost/.virtualenvs/TOX/lib/python2.7/
./home/novapost/.virtualenvs/TOX/lib/python2.7/site-packages/
./home/novapost/.virtualenvs/TOX/lib/python2.7/site-packages/workalendar-0.4.3-py2.7.egg-info/
./home/novapost/.virtualenvs/TOX/lib/python2.7/site-packages/workalendar-0.4.3-py2.7.egg-info/not-zip-safe
./home/novapost/.virtualenvs/TOX/lib/python2.7/site-packages/workalendar-0.4.3-py2.7.egg-info/PKG-INFO
./home/novapost/.virtualenvs/TOX/lib/python2.7/site-packages/workalendar-0.4.3-py2.7.egg-info/requires.txt
./home/novapost/.virtualenvs/TOX/lib/python2.7/site-packages/workalendar-0.4.3-py2.7.egg-info/dependency_links.txt
./home/novapost/.virtualenvs/TOX/lib/python2.7/site-packages/workalendar-0.4.3-py2.7.egg-info/SOURCES.txt
./home/novapost/.virtualenvs/TOX/lib/python2.7/site-packages/workalendar-0.4.3-py2.7.egg-info/top_level.txt
./home/novapost/.virtualenvs/TOX/lib/python2.7/site-packages/workalendar/
./home/novapost/.virtualenvs/TOX/lib/python2.7/site-packages/workalendar/oceania.py
./home/novapost/.virtualenvs/TOX/lib/python2.7/site-packages/workalendar/america.pyc
./home/novapost/.virtualenvs/TOX/lib/python2.7/site-packages/workalendar/usa.pyc
./home/novapost/.virtualenvs/TOX/lib/python2.7/site-packages/workalendar/europe.pyc
./home/novapost/.virtualenvs/TOX/lib/python2.7/site-packages/workalendar/core.pyc
./home/novapost/.virtualenvs/TOX/lib/python2.7/site-packages/workalendar/usa.py
./home/novapost/.virtualenvs/TOX/lib/python2.7/site-packages/workalendar/europe.py
./home/novapost/.virtualenvs/TOX/lib/python2.7/site-packages/workalendar/oceania.pyc
./home/novapost/.virtualenvs/TOX/lib/python2.7/site-packages/workalendar/canada.pyc
./home/novapost/.virtualenvs/TOX/lib/python2.7/site-packages/workalendar/asia.pyc
./home/novapost/.virtualenvs/TOX/lib/python2.7/site-packages/workalendar/canada.py
./home/novapost/.virtualenvs/TOX/lib/python2.7/site-packages/workalendar/__init__.pyc
./home/novapost/.virtualenvs/TOX/lib/python2.7/site-packages/workalendar/core.py
./home/novapost/.virtualenvs/TOX/lib/python2.7/site-packages/workalendar/africa.py
./home/novapost/.virtualenvs/TOX/lib/python2.7/site-packages/workalendar/america.py
./home/novapost/.virtualenvs/TOX/lib/python2.7/site-packages/workalendar/__init__.py
./home/novapost/.virtualenvs/TOX/lib/python2.7/site-packages/workalendar/africa.pyc
./home/novapost/.virtualenvs/TOX/lib/python2.7/site-packages/workalendar/asia.py
```

Thanks

**Comment by brunobord** [comment]
Mmmmm that's weird. I'm AFK for a few days but i'll try to investigate as
soon as possible.
Sorry for the inconvenience.

**Comment by Natim** [comment]
I tryied to reproduce without success: pip install workalendar

```
Downloading workalendar-0.4.3.tar.gz
Successfully installed lunardate-0.1.5 pyCalverter-1.6.1 pyephem-3.7.6.0 python-dateutil-2.5.3 pytz-2016.4 six-1.10.0 workalendar-0.4.3
```

Could you try again maybe?

**Comment by Natim** [comment]
However downloading the file you specified and running:

```
$ pip install workalendar-0.4.3.linux-x86_64.tar.gz
Processing workalendar-0.4.3.linux-x86_64.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
    IOError: [Errno 2] No such file or directory: '~/TMP/pip-s63TfQ-build/setup.py'
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in ~/TMP/pip-s63TfQ-build/
```

Did reproduced the matter.

**Comment by Natim** [comment]
I am not sure why you are not downloading the same file as I. Can you try to update pip and install it again? In the meantime we should probably delete the broken file from pypi.

**Comment by Natim** [comment]
Try using : https://pypi.python.org/packages/43/24/b5f63509f31dd43d9e07e596b619bbf25f10611456e068dcb075a887cfe8/workalendar-0.4.3.tar.gz#md5=058c3bc935fb180a5c0a7ad5e45fa240

**Comment by brunobord** [comment]
I've completely screwed up the PyPI 0.4.3 files.
I'm deeply sorry for that, I thought it was possible to replace a file in a release.
I've managed to release a new version: 0.4.5. Please test this and tell me if this has solved your issue.

**Comment by guilleJB** [comment]
Hi,
It's resolved with pip version 1.3.1
```bash 
pip --version
pip 1.3.1 from /home/gjulia/ve/tmp-4f5d92f7f7a8c090/lib/python2.7/site-packages/pip-1.3.1-py2.7.egg (python 2.7)
```
I'm running `pip install workalendar` and the message is: 
`Successfully installed workalendar python-dateutil lunardate pytz pyCalverter pyephem six`

So thanks @brunobord
